src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/CourtReport.java

```diff
@@ -131,4 +131,8 @@ public class CourtReport {
 
     @Column(name = "PENDING_TRANSFER")
     private Long pendingTransfer;
+
+    @JoinColumn(name = "DELIVERED_REPORT_REASON_ID", referencedColumnName = "STANDARD_REFERENCE_LIST_ID", insertable = false, updatable = false)
```

Comment:
I think this is the wrong column, it should be `DELIVERED_COURT_REPORT_TYPE_ID` referencing `R_COURT_REPORT_TYPE` shouldn't it?